azure-pipelines.yml

```diff
@@ -74,23 +74,10 @@ jobs:
           filePath: yarn-ci.sh
         displayName: yarn
 
-      ## only do scoped bundle with PRs
       - script: |
-          yarn bundle --no-cache --since $(targetBranch)
+          yarn run:published build --production --verbose --no-cache &&
```

Comment:
This part is a TEMPORARY change to test out the new bundling commands. The PR is marked as do not merge until this is removed.